.github/workflows/release.yaml

```diff
@@ -29,11 +29,11 @@ jobs:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - name: Build and Test
         run: npm ci
-      # - run: npm publish --access public
-      #   env:
-      #     NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-      - name: Publish Release
-        run: npx semantic-release --dry-run
+      - name: Create release and publish to NPM
```

Comment:
Running `changeset version` is an important part of the workflow - it consumes the changesets you've commited and bumps the version in package file accordingly.
https://github.com/changesets/changesets/blob/main/docs/intro-to-using-changesets.md#versioning-and-publishing